src/frontend/components/Felleskomponenter/FamilieAlert/FamilieAlert.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+import styled from 'styled-components';
+
+import { Alert, AlertProps } from '@navikt/ds-react';
+
+interface AlertStripeProps extends AlertProps {
+    dynamisk?: boolean;
+    className?: '';
+}
+
+const StyledAlert = styled(Alert)`
+    p {
+        margin: 0;
+    }
+`;
+
+const FamilieAlert: React.FC<AlertStripeProps> = ({
+    variant,
+    inline = true,
```

Comment:
Hva tenker du om at den har `inline = false` som default, slik at man heller definerer det opp dersom den skal være inline? Da vil vi gå fra å overstyre den når den skal bli false `<FamilieAlert inline={false}>` til å overstyre den til å bli true `<FamilieAlert inline>`

Reply:
Det er sånn `inline` er definert opp for Alert i Aksel https://aksel.nav.no/komponenter/core/alert#h475b7c370d58

Reply:
Synes måten du beskriver er ryddigere! I begynnelsen hadde vi omtrent kun alerts som var inline. Vi kan jo gjøre en vurdering om det er bedre å endre dette igjen til slik det er i Aksel, nå som vi har et par alerts som ikke er inline.